mousestyles/path_diversity/path_features_advanced.py

```diff
@@ -0,0 +1,297 @@
+from __future__ import print_function, absolute_import, division
+import numpy as np
+import pandas as pd
+# from mousestyles.path_features import path_features
```

Comment:
Just making sure, you will be using the line that's been commented out in the final version right? Also, since all function files are under the path_diversity folder, should it be `from mousestyles.path_diversity import path_features` instead?